README.md

```diff
@@ -1,34 +1,53 @@
 # Cloud Code for Visual Studio Code
 
-Cloud Code for VS Code brings the power and convenience
-of IDEs to developing cloud-native Kubernetes and Cloud Run applications.
-Cloud Code works with Google’s command-line [container tools][1] like [`skaffold`][2]
-and [`kubectl`][3] under the hood, providing local, continuous feedback on your project
-as you build, edit, and run your applications on your workstation or in the cloud.
+Cloud Code for VS Code brings the power and convenience of Google Cloud to Visual Studio Code. Our extension works
+with Google’s command-line [container tools][1] like [`skaffold`][2] and [`kubectl`][3] under the hood, providing
+local, continuous feedback on your project as you build, edit, and run your applications on your workstation or in
+the cloud. You can easily find and manage Google Cloud APIs to use. Cloud Code for VS Code even helps you connect to
+Google Cloud resources while running on Kubernetes and emulated Cloud Run locally.
 
-## New App, Deploy and Debug
+## App, Deploy and Debug
 
-With Cloud Code, you can create a new application, deploy it to a remote Kubernetes
-cluster, and debug your running app in seconds!
+With Cloud Code, you can create a new Kubernetes or Cloud Run application, deploy it to a locally or remotely in
+seconds with live updates as you write code. Or you can seamlessly debug Cloud Run applications locally as well as
+Kubernetes applications both locally and remotely.
 
 ![New App, Deploy and Debug][12]
 
+## Google Cloud API library browser
+
+Cloud Code allows you to find Google Cloud APIs without leaving your IDE and breaking your development flow. Easily
+add these APIs to your project and discover more with our API library browser.
+
+![Client Library Explorer][14]
+
+## Manage your secrets with Secrets Manager
+
+With Cloud Code get the convenience of managing your application secrets directly in VS Code. Quickly connect to
+Google Cloud when running in your IDE locally even while debugging or testing your Kubernetes and Cloud Run
+applications.
+
+![Secret Manager][15]
+
 ## Key Features
```

Comment:
missing buildpacks

Reply:
How about

## Use Google Cloud buildpacks

With Cloud Code you can focus on building your application, not containerizing it by using Google Cloud buildpacks.
Easily deploy your code directly to GKE or Cloud Run without worrying about a Dockerfile.